pyaedt/edb_core/padstack.py

```diff
@@ -496,8 +500,12 @@ def create_coax_port(self, padstackinstance, use_dot_separator=True):
             port_name = "{0}_{1}_{2}".format(cmp_name, pin_name, net_name)
         if not padstackinstance.IsLayoutPin():
             padstackinstance.SetIsLayoutPin(True)
-
         res = padstackinstance.GetLayerRange()
+        if user_defined_port_name:
+            if not self._is_port_exist(user_defined_port_name):
+                port_name = user_defined_port_name
+        else:
+            self._logger.info("Port name already assigned on existing port, falling back to regular naming convention")
```

Comment:
Perhaps the following:
"An existing port is already assigned this name. Resetting to the default name."